afl-fuzz.c

```diff
@@ -1258,6 +1258,13 @@ static void minimize_bits(u8* dst, u8* src) {
 
 }
 
+double rand_double(double min, double max) 
```

Comment:
@jiradeto I think we will not need the `min` and `max`. Maybe we can simplify it like this:

```C
// return random value in interval [0.0,1.0)
double rnd = (double)UR(RAND_MAX);
double max = (double)RAND_MAX;
return rnd / max;
```

Reply:
Ok, I was thinking we may need to use this double random function in the future for different purpose so that's why the function requires min/max. But yeah we may not need it for now. Fixed :)

Reply:
@jiradeto Yeah, I understand. :) Thanks for the fix!